.github/workflows/s8s4_linear_cutlass_test.yml

```diff
@@ -0,0 +1,43 @@
+name: Run CUTLASS-based W4A8 kernel tests
+
+on:
+  push:
+    branches:
+      - w4a8-cutlass
```

Comment:
I think you can enable this more generally to prevent regressions: https://github.com/pytorch/ao/blob/abff563ba515576fc48cd4ac0feb923dd65dc267/.github/workflows/pr-label-check.yml#L3